internal/credential/vault/testing.go

```diff
@@ -0,0 +1,47 @@
+package vault
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/db"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCredentialStores creates count number of vault credential stores in
+// the provided DB with the provided scope id. If any errors are
+// encountered during the creation of the credential stores, the test will
+// fail.
+func TestCredentialStores(t *testing.T, conn *gorm.DB, wrapper wrapping.Wrapper, scopeId string, count int) []*CredentialStore {
+	t.Helper()
+	assert, require := assert.New(t), require.New(t)
+	w := db.New(conn)
+	var css []*CredentialStore
+	for i := 0; i < count; i++ {
+
+		// cs, err := NewAccount(authMethodId, WithLoginName(fmt.Sprintf("token%d", i)))
```

Comment:
You still need this for something, seems it was replaced with the next line

Reply:
ty!